brev-bestilling-boot/src/test/java/no/nav/pensjon/brev/bestilling/ApplicationTest.java

```diff
@@ -9,25 +9,21 @@
 import org.apache.kafka.clients.consumer.ConsumerConfig;
 import org.apache.kafka.clients.consumer.ConsumerRecords;
 import org.apache.kafka.common.serialization.ByteArrayDeserializer;
-import org.apache.kafka.common.serialization.ByteArraySerializer;
 import org.junit.BeforeClass;
 import org.junit.ClassRule;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.kafka.core.DefaultKafkaConsumerFactory;
-import org.springframework.kafka.core.DefaultKafkaProducerFactory;
-import org.springframework.kafka.core.KafkaTemplate;
 import org.springframework.kafka.test.rule.EmbeddedKafkaRule;
 import org.springframework.kafka.test.utils.KafkaTestUtils;
 import org.springframework.test.context.junit4.SpringRunner;
 
 @RunWith(SpringRunner.class)
 @SpringBootTest
-public class StreamBinderTest {
+public class ApplicationTest {
 
-    private static final String INPUT_TOPIC = "BREV_BESTILLING";
-    private static final String OUTPUT_TOPIC = "BREV_ER_BESTILLT";
+    private static final String OUTPUT_TOPIC = "pensjonsbrev.pensjon-brev-bestilling";
```

Comment:
Bør det ikke vare "brev_er_bestilt" topic navn ? Så at den er det samme som eventen. Topic "pensjon-brev-bestilling" høres litt rar for meg.

Reply:
Her har jeg bare brukt det topic som allerede er definert: https://github.com/navikt/pensjon-brev-bestilling/blob/master/topics/pensjon-brev-bestilling.yaml#L4
Men jeg er enig i at "brev_er_bestilt" er bedre. Endrer det.

Reply:
Det er bare nå jeg lagd merke på det.

Reply:
```
Deployment failed: resource 2: Topic.kafka.nais.io "brev_er_bestilt" is invalid: metadata.name: Invalid value: "brev_er_bestilt": a DNS-1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*') (total of 1 errors)
```
Kan ikke bruke "_"